apps/web-mzima-client/src/app/post/post-head/post-head.component.html

```diff
@@ -2,10 +2,11 @@
   <mzima-client-button
     fill="clear"
     color="gray"
-    tabindex="-1"
+    ariaLabel="Edit this post"
     *ngIf="editable"
     [iconOnly]="true"
     (buttonClick)="editPost($event)"
+    (keydown.enter)="editPost($event)"
     class="post-head__control"
```

Comment:
Can you explain a little more about this change?

Reply:
@Angamanga this line here listens for keyboard enter event on the edit icon button when navigating with the keyboard using tab.